pkg/validation/validate_cluster.go

```diff
@@ -83,6 +86,16 @@ func ValidateCluster(clusterName string, instanceGroupList *kops.InstanceGroupLi
 		return nil, fmt.Errorf("Cannot get nodes for %q: %v", clusterName, err)
 	}
 
+	validationCluster.ComponentsHealthy, err = collectComponentStatus(clusterKubernetesClient)
+	if err != nil {
+		return nil, fmt.Errorf("Cannot get component status for %q: %v", clusterName, err)
```

Comment:
Nit: we tend to start these with a lower case letter, for chaining.  i.e. "cannot get..."